default.yaml

```diff
@@ -72,6 +72,10 @@ test_accounts:
 
 sms_provider: "twilio"
 
+geetest:
```

Comment:
I think these should be in [this file](https://github.com/GaloyMoney/galoy/blob/main/charts/galoy/templates/deployment.yaml). @nicolasburtey

Reply:
yes in addition to what we have here